### PR TITLE
Ensure that the Spaces list has focus in the Add Space wizard

### PR DESCRIPTION
**Pull request by erikjv**
And that the list is also tab-focusable.

### CODE REVIEW

File: src/gui/folderwizard/spacespage.ui [hunk 1]
```diff
@@ -26,6 +26,9 @@
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
+     <property name="focusPolicy">
+      <enum>Qt::WheelFocus</enum>
```
Comment (TheOneRing) [inline src/gui/folderwizard/spacespage.ui:30]: Strong focus?

File: src/gui/spaces/spacesbrowser.ui [hunk 2]
```diff
@@ -27,7 +27,11 @@
     <number>0</number>
    </property>
    <item>
-    <widget class="OCC::QmlUtils::OCQuickWidget" name="quickWidget" native="true"/>
+    <widget class="OCC::QmlUtils::OCQuickWidget" name="quickWidget" native="true">
+     <property name="focusPolicy">
+      <enum>Qt::WheelFocus</enum>
```
Comment (TheOneRing) [inline src/gui/spaces/spacesbrowser.ui:32]: Strong focus?

**Review by TheOneRing — CHANGES_REQUESTED** [review]

**Comment by erikjv** [comment]
I'm going to borrow a wheel mouse to test the difference

File: src/gui/spaces/spacesbrowser.ui (hunk 2 above)
Comment (erikjv) [reply src/gui/spaces/spacesbrowser.ui]: Yes, strong focus is enough

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**